README.md

```diff
@@ -108,101 +108,86 @@ fetch('<url>', {
+If there are no tokens available for the given issuer, the returned promise rejects with an error. Otherwise, it invokes the PrivacyPass redemption protocol against the issuer, with the token (potentially, if specified by an extension, along with associated redemption metadata) attached in the Sec-Trust-Token request header and the resulting Redemption Record (RR) being expected in the Sec-Trust-Token response header.
 
-The structure of the Signed Redemption Record (SRR) is:
+The RR is HTTP-only and JavaScript is only able to access/send the RR via Trust Token Fetch APIs. It is also cached in new first-party storage accessible only by these APIs for subsequent visits to that first-party.
 
-
-```
-{
-  Redemption timestamp,
-  ClientData: {
-    Publisher Origin
-  },
-  Metadata: {
-    Trust Token Key ID
-  }
-  Signature of the above verifiable by well-known public key of the issuer,
-  optional expiry timestamp
-}
-```
-
-
-The redemption timestamp is included to ensure the freshness of the SRR. The `ClientData` comes from the client as part of the redemption request and includes the publisher the redemption occurred on. The `Metadata` includes the key ID, so that consumers of the SRR can compare against the currently active key commitment.
-
-The SRR is HTTP-only and JavaScript is only able to access/send the SRR via Trust Token Fetch APIs. It is also cached in new first-party storage accessible only by these APIs for subsequent visits to that first-party. The expiry is a recommendation to the UA, telling it when to perform a refresh of the SRR. The UA is allowed to choose an expiry beyond that specified with the SRR.
-
-To mitigate [token exhaustion](#trust-token-exhaustion), a site can only redeem tokens for a particular issuer if they have no cached non-expired SRRs from that issuer or if they are the same origin as the issuer and have set the `refresh` parameter.
+To mitigate [token exhaustion](#trust-token-exhaustion), a site can only redeem tokens for a particular issuer if they have no cached non-expired RRs from that issuer or if they are the same origin as the issuer and have set the `refresh` parameter.
 
 
 ### Forwarding Redemption Attestation
 
-Signed Redemption Records are only accessible via a new option to the Fetch API:
+Redemption Records are only accessible via a new option to the Fetch API:
 
 
 ```
 fetch(<resource-url>, {
   ...
   trustToken: {
-    type: 'send-srr',
+    type: 'send-rr',
     issuer: <issuer>
   }
   ...
 });
 ```
 
 
-The SRR will be added as a new request header `Sec-Signed-Redemption-Record`. This option to Fetch is only usable in the top-level document. If there are no SRRs available, the request header will be omitted.
+The RR will be added as a new request header `Sec-Redemption-Record`. This option to Fetch is only usable in the top-level document. If there are no RRs available, the request header will be omitted.
```

Comment:
If there are no RRs available, the request header will be empty rather than omitted (in our current impl)